src/components/Tooltip/TooltipExamples.module.scss

```diff
@@ -24,3 +24,9 @@
   align-items: center;
   height: 200px;
 }
+
+.CustomTipExample {
+  background-color: #0d1f40;
```

Comment:
I didn't bother using nora variables as this is just a custom example (even though this is the color we want 👅 )